core/templates/pages/exploration-editor-page/translation-tab/services/translation-language.service.ts

```diff
@@ -41,6 +41,11 @@ export class TranslationLanguageService {
     return this.activeLanguageCode;
   }
 
+  getActiveLanguageDirection(): string {
```

Comment:
Can you please check whether this is working for languages like `Hinglish` which is not a content language (it's only in the translation language list.)

Reply:
Hinglish only appears in the `SUPPORTED_AUDIO_LANGUAGES` constant,
Should I add it to the `SUPPORTED_CONTENT_LANGUAGES` constant?

Reply:
No, as Hinglish is not a content language i.e, we don't allow creating exploration in the Hinglish language. I would suggest having directions in the AUDIO_LANGUAGES as well (though it doesn't sound correct to have a direction for the audio language I think we use the audio language list for translation as well so for the translations we do need the direction.)

Reply:
I'm also going to update getActiveLanguageDirection to search through `SUPORTED_AUDIO_LANGUAGES` as well,
does that make sense?

Reply:
if the function is in translation-language.service.ts then only use `SUPORTED_AUDIO_LANGUAGES`. (Make sure to add a comment explaining that currently we only allow translation in the supported voiceover/audio languages.)

Reply:
The function referencing the language constants is actually in language-util.service.ts, so I separated it into two functions, one for audio languages, one for content languages. The function in translation-language.service.ts only uses the audio languages function from language-util.service.ts.